tools/postgres/schema.sql

```diff
@@ -301,6 +303,7 @@ CREATE TABLE webknossos.users(
   userConfiguration JSONB NOT NULL,
   isDeactivated BOOLEAN NOT NULL DEFAULT false,
   isAdmin BOOLEAN NOT NULL DEFAULT false,
+  isOrganizationOwner BOOLEAN NOT NULL DEFAULT false,
```

Comment:
I had a look at this as well, but there are apparently a few downsides to the unique index method, see the comments here https://stackoverflow.com/a/16236566/7564727 – This might not be a problem for us, but since we don’t have a hard requirement for just a single owner, I’d leave it as is for now.